export_test.go

```diff
@@ -1,5 +1,22 @@
 package tarantool
 
+import (
+	"net"
+	"time"
+)
+
 func (schema *Schema) ResolveSpaceIndex(s interface{}, i interface{}) (spaceNo, indexNo uint32, err error) {
 	return schema.resolveSpaceIndex(s, i)
 }
+
+// Yes, this is an anti-pattern. We will test implementation, not public API.
+// But in this case it is important to do it. Because we must to test
+// an integration with an SSL library.
```

Comment:
Isn't go `_test` packages have access to private (lowercase) methods?

Reply:
No, it's kind of valid hack. So you can to export internals, see https://go.dev/src/net/http/export_test.go . But it's better to avoid it.

Reply:
I think it would be better to delete the obvious comment (we don't want to share the internal implementation, it's a hack etc). It's just confusing and has no real value. I deleted it.